service/src/main/java/uk/gov/mca/beacons/api/legacybeacon/rest/LegacyBeaconController.java

```diff
@@ -39,4 +45,19 @@ public WrapperDTO<LegacyBeaconDTO> findById(@PathVariable("uuid") UUID id) {
 
     return legacyBeaconMapper.toWrapperDTO(legacyBeacon);
   }
+
+  @PatchMapping(value = "/backoffice/{uuid}/delete")
+  public ResponseEntity<Void> delete(
+    @PathVariable("uuid") UUID beaconId,
+    @RequestBody @Valid DeleteBeaconDTO dto
+  ) {
+    if (
+      !beaconId.equals(dto.getBeaconId())
+    ) throw new InvalidBeaconDeleteException();
+
+    // yuck
+    registrationService.deleteLegacyBeacon(dto);
```

Comment:
'yuck' because I'm not sure if deleteLegacyBeacon() should live in the registrationService. Feels weird to invoke a registrationService from a LegacyBeaconController

Reply:
The controllers seem to be based around functionality, so a deletecontroller that manages legacy and modern would probably work nicely